compute/disks/disks_test.go

```diff
@@ -156,18 +135,23 @@ func TestComputeDisksSnippets(t *testing.T) {
 
 	buf := &bytes.Buffer{}
 
-	t.Run("createDiskSnapshot and deleteDisk", func(t *testing.T) {
-		err := createDisk(ctx, tc.ProjectID, zone, diskName, sourceImage)
+	t.Run("Create zonal and regional disks from a snapshot and delete them", func(t *testing.T) {
```

Comment:
Does this test need to be run concurrently with the following tests? If not, I'd break these into separate test cases. The `T.Run()` method is intended for testing concurrency.

Reply:
I'm afraid the correctness of tests goes hand-in-hand with publishing samples. See the [Samples Guide](https://googlecloudplatform.github.io/samples-style-guide/#testing).

Please refactor into separate test cases.